docs/source/conf.py

```diff
@@ -27,11 +27,12 @@ class DocBuildError(Exception):
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 root_path = os.path.abspath(os.path.join("..", ".."))
 sys.path.insert(0, root_path)
-import commands
-_sphinx_apidoc = commands.getoutput('which sphinx-apidoc').strip()
+from avocado.utils import process
```

Comment:
use process.system_output instead of process.run().
One demo from avocado follow as below:

********************************************
-        replace_output = commands.getoutput(replace_cmd)
-        cat_output = commands.getoutput(cat_cmd)
+        replace_output = process.system_output(replace_cmd, shell=True)
+        cat_output = process.system_output(cat_cmd, shell=True)